src/main/java/com/icthh/xm/ms/configuration/config/AppMetricsConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.icthh.xm.ms.configuration.config;
+
+import com.codahale.metrics.MetricRegistry;
```

Comment:
why you added this class? 
Please configure metrics as it described on wiki: https://github.com/xm-online/xm-online/wiki/How-to-create-new-XM-microservice-like-project#configure-xm-metrics

Reply:
This configuration class was added because metricRegistry must be wired while starting the microservice. This microvervice doesn't start without this configuration. 
This microservice doesn't contain `HikariDataSource` that's why I spkipped it.

Reply:
please folow the instruction without step with HikariDataSource
also look to the class
`com.icthh.xm.commons.metric.MetricsConfiguration`